ambry-store/src/test/java/com.github.ambry.store/CuratedLogIndexState.java

```diff
@@ -66,6 +66,7 @@
   private static final long HARD_DELETE_LAST_PART_SIZE = 13;
 
   static final DiskIOScheduler DISK_IO_SCHEDULER = new DiskIOScheduler(null);
+  static final DiskSpaceAllocator DISK_SPACE_ALLOCATOR = StoreTestUtils.getDiskSpaceAllocator();
```

Comment:
can't we statically declare DISK_SPACE_ALLOCATOR in StoreTestUtils and statically import in every test class